packages/ui/src/working-groups/components/WorkingGroupListItem.tsx

```diff
@@ -35,9 +39,16 @@ export function WorkingGroupListItem({ group }: WorkingGroupProps) {
       <GroupImageContainer as={GhostRouterLink} to={groupAddress}>
         <WorkingGroupImage groupName={group.name} />
       </GroupImageContainer>
-      <GroupContentBlock as={GhostRouterLink} to={groupAddress}>
-        <GroupTitle>{nameMapping(group.name)}</GroupTitle>
-        <GroupContent>{subtitleMapping(group.name)}</GroupContent>
+      <GroupContentBlock onMouseEnter={() => setShowTooltip(true)} onMouseLeave={() => setShowTooltip(false)}>
```

Comment:
There's a [failing test](https://github.com/Joystream/pioneer/actions/runs/3271941116/jobs/5382339994) now, could it be related? `test/app/pages/WorkingGroup.test.tsx`
>  |     > 33 |     await waitForElementToBeRemoved(() => loaderSelector(true), {})